Ops/XConsole.cs

```diff
@@ -107,28 +106,28 @@ public static void WriteLine(string value, IConsoleColorSet colorSet = null, boo
 
         public static void Progress(string stepDescription, int progress, int total)
         {
-            int totalChunks = 30;
+            const int TOTALCHUNKS = 30;
 
             //draw empty progress bar
             Console.CursorLeft = 0;
             Console.Write("["); //start
-            Console.CursorLeft = totalChunks + 1;
+            Console.CursorLeft = TOTALCHUNKS + 1;
             Console.Write("]"); //end
             Console.CursorLeft = 1;
 
             double pctComplete = Convert.ToDouble(progress) / total;
-            int numChunksComplete = Convert.ToInt16(totalChunks * pctComplete);
+            int numChunksComplete = Convert.ToInt16(TOTALCHUNKS * pctComplete);
 
             //draw completed chunks
             Console.BackgroundColor = ConsoleColor.Green;
             Console.Write("".PadRight(numChunksComplete));
 
             //draw incomplete chunks
             Console.BackgroundColor = ConsoleColor.Gray;
-            Console.Write("".PadRight(totalChunks - numChunksComplete));
+            Console.Write("".PadRight(TOTALCHUNKS - numChunksComplete));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/app/onurakkaya/DSM.Core/pullRequest?prid=4732028)